torch/fx/experimental/proxy_tensor.py

```diff
@@ -174,7 +174,25 @@ def has_proxy_slot(obj: Tensor, tracer: _ProxyTracer) -> bool:
     return bool(get_proxy_slot(obj, tracer, False, lambda _: True))
 
 
-_PySymProxyType = Callable[[], Proxy]
+class Thunk(Generic[R]):
+    f: Optional[Callable[[], R]]
+    r: Optional[R]
+
+    __slots__ = ['f', 'r']
+
+    def __init__(self, f: Callable[[], R]):
+        self.f = f
+        self.r = None
+
+    def force(self) -> R:
```

Comment:
Why .force() vs `.__call__()` ?

Reply:
I want to be able to easily able to grep force sites where I force the thunk